lib/apiservers/engine/backends/container.go

```diff
@@ -64,32 +63,32 @@ const (
 
 // ContainerExecCreate sets up an exec in a running container.
 func (c *Container) ContainerExecCreate(config *types.ExecConfig) (string, error) {
-	return "", fmt.Errorf("%s does not implement container.ContainerExecCreate", c.ProductName)
+	return "", fmt.Errorf("%s does not implement container.ContainerExecCreate", ProductName())
```

Comment:
@sflxn All of these should be pulled out into a message file - it shouldn't take long. The intent is that there is an actual github issue link included for unimplemented features.